crates/node_binding/src/resolver_factory.rs

```diff
@@ -21,8 +21,8 @@ pub struct JsResolverFactory {
 #[napi]
 impl JsResolverFactory {
   #[napi(constructor)]
-  pub fn new() -> napi::Result<Self> {
-    let input_filesystem = Arc::new(NativeFileSystem {});
+  pub fn new(enable_pnp: bool) -> napi::Result<Self> {
```

Comment:
nit: No strong opinion, but I'd suggest to make an option bag struct rather than pass `enable_pnp` as a raw boolean; if you need to add other options (which for a JS resolver is conceivable), you won't have to significantly change the API.

Reply:
actually pnp support is a special case and is global setting, all other js resolverOptions is passed using https://github.com/web-infra-dev/rspack/blob/3eaae99bb796ca5487d720947dcc73bd704c8467/packages/rspack/src/ResolverFactory.ts#L23, which supports other complex resolver setting and support multi resolver Instance with different resolve options